src/components/Fields/ArrayItemDetails.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { TypeFormat, TypePrefix } from '../../common-elements/fields';
+import { ConstraintsView } from './FieldContstraints';
+import { Pattern } from './Pattern';
+import { SchemaModel } from '../../services';
+import styled from '../../styled-components';
+
+export function ArrayItemDetails({ schema }: { schema: SchemaModel }) {
+  if (!schema || (schema.type === 'string' && !schema.constraints.length)) return null;
+
+  return (
```

Comment:
Hi. This is causing that type label of arrays of any type except string is followed by `[ items ]`:
![image](https://user-images.githubusercontent.com/22495416/153569980-d06027fa-a27c-4cd7-a1b5-bec5b7b511df.png)